kodilla-stream/src/main/java/com/kodilla/stream/array/ArrayOperations.java

```diff
@@ -5,15 +5,19 @@
 public interface ArrayOperations {
 
     public static double getAverage(int[] numbers) {
-        IntStream.range(0, numbers.length)
-                .map(n -> numbers[n])
-                .forEach(System.out::println);
+        if (numbers == null) {
+            return 0;
+        } else if (numbers.length > 0) {
```

Comment:
nie potrzebne else.
Czy potrzebny warunek numbers.length > 0? Jak jest == 0, to nic się nie stanie